test/exportable/ExportableInstance.test.ts

```diff
@@ -27,4 +27,26 @@ describe('ExportableInstance', () => {
     ].join(EOL);
     expect(i.toFSH()).toBe(expectedResult);
   });
+
+  it('should export an instance with assignment rules', () => {
+    const exInstance = new ExportableInstance('MyInstance');
+    exInstance.instanceOf = 'Observation';
+
+    const statusRule = new ExportableAssignmentRule('status');
+    statusRule.value = 'final';
```

Comment:
I think that `final` is actually a `code` on Observation. I think this would be better represented using a `FSHCode` here. That would change the `expectedResult` below slightly.

Reply:
You're absolutely right! I'll change that to make it an actual code and change the example FSH as well. Nice catch!